### PR TITLE
WAVE2-758-adding new relic deploy marker

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

**Comment by nindusaini** [comment]
Yes Geoff, I have that file inside quicksilver folder. Is that not the right place???

**Comment by gsowrey** [comment]
It needs to be in the site root, along with the .probo.yml file.


On Nov 29, 2016, at 10:52, Navinder <notifications@github.com<mailto:notifications@github.com>> wrote:


Yes Geoff, I have that file inside quicksilver folder. Is that not the right place???

-

**Comment by nindusaini** [comment]
Ok, moved it to root folder.

File: pantheon.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+php_version: 7.0
+
+# Drush Version
+# https://pantheon.io/docs/pantheon-yml/#drush-version
+drush_version: 8
+
+# Protected Web Paths
+# https://pantheon.io/docs/pantheon-yml#protected-web-paths
+protected_web_paths:
+   - /example.txt
```
Comment (gsowrey) [inline pantheon.yml]: Let's make sure the example stuff is removed. ;)

File: pantheon.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+   - /example_directory
+
+# Nested Docroot
+# https://pantheon.io/docs/pantheon-yml#nested-docroot
+web_docroot: true
```
Comment (gsowrey) [inline pantheon.yml]: I think this needs to be false, we're not hosting a nested webroot. See https://pantheon.io/docs/pantheon-yml/

File: pantheon.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# https://pantheon.io/docs/pantheon-yml#quicksilver
+workflows:
+  # Log to New Relic when deploying to test or live.
+    deploy:
+      after:
+        - type: webphp
+          description: Log to New Relic
+          script: private/quicksilver/new_relic_deploy.php
+    # Also log sync_code so you can track new code going into dev/multidev.
+    sync_code:
```
Comment (gsowrey) [inline pantheon.yml]: Do we need the sync_code?

File: private/quicksilver/new_relic_deploy/new_relic_deploy.php [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  // Get the annotation:
+  $changelog = `git tag -l -n99 $deploy_tag`;
+  $user = $_POST['user_email'];
+}
+
+
+// Use New Relic's v1 curl command-line example.
+// TODO: update to use v2 API with JSON, plus curl() in PHP.
+// Blocked by needing the app_id to use v2 API
+$curl = 'curl -H "x-api-key:'. $data['api_key'] .'"';
```
Comment (gsowrey) [inline private/quicksilver/new_relic_deploy/new_relic_deploy.php]: So what does the data in New Relic look like? Are we getting data for develop, dev, test, and live? Or only live?

File: pantheon.yml (hunk 1 above)
Comment (nindusaini) [inline pantheon.yml]: Removed, but added pantheon.yml here, so its hidden from public accees

File: pantheon.yml (hunk 3 above)
Comment (nindusaini) [inline pantheon.yml]: I thought we do want to monitor how code is sync'd, so we know where/if an issue starts.

File: pantheon.yml (hunk 2 above)
Comment (nindusaini) [inline pantheon.yml]: Changed to false.

**Comment by gsowrey** [comment]
Hmm. ‘Tis a good point. Yeah, maybe leave that on for now. We can always disable it if it doesn’t prove to be useful.

**Comment by gsowrey** [comment]
What’s the data look like? Is it useful information in New Relic?

**Pull request merged**